packages/redux-toolbelt-thunk/test/makeThunkAsyncActionCreator.js

```diff
@@ -231,5 +231,22 @@ describe('Custom args mapper (argsMapper in options)', () => {
     })
     store.dispatch(customAction('arg1', 'arg2'))
   })
+
+  test('cancel previous requests', () => {
+    const store = mockStore()
+    const customAction = makeAsyncThunkActionCreator('ACTION',  id => Promise.resolve(id), null, {ignoreOlderParallelResolves: true})
+    store.dispatch(customAction(1))
+    store.dispatch(customAction(2))
```

Comment:
i think the test is expected to fail because you confused the option 
handleOlderParallelSuccesses

so i think you need to check the length of the actions